1-fundamentos/b03-intro-html-css/d02-primeiros-passos/styles.css

```
@@ -0,0 +1,22 @@
+h1 {
+    font-size: 65px;
```

```suggestion
    font-size: 66px;
```
Será que não fica legal um pouquinho maior?